src/CssInliner.php

```diff
@@ -1305,15 +1072,15 @@ private function removeUnsupportedOfTypePseudoClasses(string $selectorPart): str
      * Applies `$this->matchingUninlinableCssRules` to `$this->domDocument` by placing them as CSS in a `<style>`
      * element.
      *
-     * @param string $uninlinableCss
+     * @param CssDocument $cssDocument
      *        This may contain any `@import` or `@font-face` rules that should precede the CSS placed in the `<style>`
      *        element.  If there are no unlinlinable CSS rules to copy there, a `<style>` element will be created
-     *        containing just `$uninlinableCss`.  `$uninlinableCss` may be an empty string; if it is, and there are no
+     *        containing only the applicable at-rules from `$cssDocument`.  If there are none, and there are also no
      *        unlinlinable CSS rules, an empty `<style>` element will not be created.
```

Comment:
no unlinlinable -> only inlinable

Reply:
If it's changed to "only inlinable" I think it becomes misleading - the method only deals with uninlinable rules and never copies inlinable rules to the `<style>` element (whether or not there are any).

Reply:
Apart from the typo in "unlinlinable", isn't "no uninlinable rules" and ""only inlinable rules" semantically the same, but without the double negation?

Reply:
"If there are only inlinable CSS rules to copy there" would imply inlinable rules are copied there.  They are not.

Reply:
Ah, okay, I see.

Then let's please only fix the typo.

Reply:
Done.